src/tests/ftest/dfuse/simul.py

```diff
@@ -103,22 +106,26 @@ def run_simul(self, include=None, exclude=None, raise_exception=True):
         start_dfuse(self, dfuse, pool, container)
         dfuse.check_running()
 
-        # The use of MPI here is to run in parallel all simul tests on a single host.
-        if not load_mpi(mpi_type):
-            raise MPILoadError(mpi_type)
-
-        # Run simul
         simul_cmd = os.path.join(simul_path, "simul")
-        if include and not exclude:
-            cmd = "{0} -vv -d {1} -i {2}".format(simul_cmd, dfuse.mount_dir.value, include)
-        elif exclude and not include:
-            cmd = "{0} -vv -d {1} -e {2}".format(simul_cmd, dfuse.mount_dir.value, exclude)
+        self.log_step(f"Running simul on {mpi_type}")
+        if faillist is None:
+            mpirun = Mpirun(f"{simul_cmd} -vv -d {dfuse.mount_dir.value} -e {exclude}",
+                            mpi_type=mpi_type)
+            mpirun.assign_hosts(get_local_host(), dfuse.mount_dir.value)
+            mpirun.assign_processes(processes=num_proc, ppn=ppn)
```

Comment:
Here if you just give `ppn` the function will calculate `np` based on the number of hosts already assigned above
```suggestion
            mpirun.assign_processes(ppn=ppn)
```

Reply:
Thank you! Fixed as suggested.